Tests/TuistCoreTests/Graph/GraphDependencyReferenceTests.swift

```diff
@@ -3,16 +3,33 @@ import TSCBasic
 import TuistSupport
 import XCTest
 
+import SwiftUI
```

Comment:
Why is this necessary?

Reply:
it absolutely is not! good spot 🤦‍♂️

I was using Xcode 13 beta and it auto imports frameworks based on types used it must have thought I was using a SwiftUI type and I completely missed this. Will revert this!